tests/unittest.py

```diff
@@ -620,18 +620,18 @@ def register_appservice_user(
         self,
         username: str,
         appservice_token: str,
-    ) -> str:
+    ) -> Tuple[str, str]:
         """Register an appservice user as an application service.
         Requires the client-facing registration API be registered.
 
         Args:
             username: the user to be registered by an application service.
-                Should be a full username, i.e. ""@localpart:hostname" as opposed to just "localpart"
+                Should NOT be a full username, i.e. just "localpart" as opposed to "@localpart:hostname"
             appservice_token: the acccess token for that application service.
 
         Raises: if the request to '/register' does not return 200 OK.
 
-        Returns: the MXID of the new user.
+        Returns: the MXID of the new user, the device ID of the new user's first device.
```

Comment:
```suggestion
        Returns:
            The MXID of the new user, the device ID of the new user's first device.
```

Reply:
ah deary me how did I forget to update that. Thanks! :)